cpp/src/arrow/util/decimal_test.cc

```diff
@@ -384,10 +389,10 @@ static const ToStringTestData kToStringTestData[] = {
     {-1234567890123456789LL, 25, "-1.234567890123456789E-7"},
 };
 
-class Decimal128ToStringTest : public ::testing::TestWithParam<ToStringTestData> {};
+class Decimal128ToStringTest : public ::testing::TestWithParam<ToStringTestParam> {};
 
 TEST_P(Decimal128ToStringTest, ToString) {
-  const ToStringTestData& data = GetParam();
+  const ToStringTestParam& data = GetParam();
```

Comment:
How about changing local variable name to `param` from `data` too?

Reply:
Done.